app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the welcome controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
Likely, for you, it will be since I typically encourage students to import stylesheets (like bootstrap).